cli/src/utils/build-stitched-schema.ts

```diff
@@ -0,0 +1,18 @@
+import { promises as fs } from 'fs';
+import * as globby from 'globby';
+import { ResourceMetadataInput, SchemaInput } from '../client';
+
+interface SchemaFilesInput {
+  metadata: ResourceMetadataInput;
+  schemaFiles: string[];
+}
+
+export default async function ({ metadata, schemaFiles }: SchemaFilesInput): Promise<SchemaInput> {
```

Comment:
I'd change the name of this file, I thought it was doing some graphql federation/schema stitching work
Maybe `readSchemaFiles` or something like that? It's just concatting some files together

Reply:
Fixed